scripts/zones/RuLude_Gardens/npcs/_6r2.lua

```diff
@@ -0,0 +1,57 @@
+-----------------------------------
+-- Area: Ru'Lude Gardens
+-- Door: Bastokan Emb.
+-- Bastok Missions 3.3 "Jeuno" and 4.1 "Magicite"
+-----------------------------------
+local ID = require("scripts/zones/RuLude_Gardens/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    local pNation = player:getNation()
+
+    if pNation == dsp.nation.BASTOK then
+        local currentMission = player:getCurrentMission(pNation)
+        local MissionStatus = player:getVar("MissionStatus")
```

Comment:
is MissionStatus needed? only see it being used once here 🐱

Reply:
while that is true, ima let it stay as is for width reasons